torch_np/_ndarray.py

```diff
@@ -370,7 +370,7 @@ def reshape(self, *shape, order="C"):
 
     def sort(self, axis=-1, kind=None, order=None):
         # ndarray.sort works in-place
-        self.tensor.copy_(_funcs._sort(self.tensor, axis, kind, order))
+        self.tensor.copy_(_funcs_impl._sort(self.tensor, axis, kind, order))
```

Comment:
TBH, I'm not a fan of hardcoding the top-level module name here. https://github.com/Quansight-Labs/numpy_pytorch_interop/issues/86#issuecomment-1479524808 plans that `_ndarray.py` depends on pretty much everything, so it does here. And imports form a DAG, which is nice I think :-).

Re normalizing the `axis` arguments. The current `operator.index(axis)` normalization of zero-dim arrays is a guard for a handful of usages in the numpy test suite. I'd guess it's very niche usage. A more general thing to do with all the axis arguments would be to centralize `normalize_axis_tuple` calls. This requires adding axis tuple support to a bunch of pytorch functions (this not consistent in pytorch), so I'd rather push it to later.

Reply:
Yes, sorry, I meant:
```
_funcs.copyto(self, _funcs.sort(self, axis, kind, order))
```
Otherwise, the current code just happens to work because we don't need to apply `normalize`. If that happened to change in the future, this function would break.

Also, as a small note, the `_sort` helper function is no longer necessary.

Reply:
Done in https://github.com/Quansight-Labs/numpy_pytorch_interop/pull/105/commits/8b75e3fde5f9911b515a5bfec6f2814e16d68056